app/scss/style.scss

```diff
@@ -0,0 +1,233 @@
+$color-purple: #a26ded;
+$color-green: #65a21f;
+$color-darkgreen: #3b7b1d;
+$color-gray: #e9ede7;
+$color-darkgray: #444444;
+$color-white: #ffffff;
+
+$color-list-lightgreen: #d1e8ba;
+$color-list-darkgreen: #96b680;
+$color-list-lightpink: #ffdedd;
+$color-list-darkpink: #ce8b82;
+
+$color-text: #333;
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 100%;
+}
+
+body {
+  background: $color-white;
+  background-size: cover;
+  color: $color-text;
+  font-family: Arial, "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+  line-height: 1.5em;
+  text-align: center;
+}
+
+:focus { outline: 0; }
+
+.clearfix::before,
+.clearfix::after {
+  content: "";
+  display: table;
+}
+
+.clearfix::after {
+  clear: both;
+}
+
+.clearfix {
+  zoom: 1; // ie 6/7
+}
+
+
+[role="banner"] {
+  background: $color-green;
+  border-bottom: 4px $color-darkgreen solid;
+
+  div {
+    color: $color-white;
+    font-size: 36px;
+    font-weight: bold;
+    margin: 0 auto;
+    padding: 20px 0;
+    text-align: left;
+    vertical-align: middle;
+    width: 1000px;
+  }
+}
+
+.selectbar {
+  background: $color-gray;
+  margin-bottom: 20px;
+  padding: 20px 0;
+}
+
+select {
+  -webkit-appearance: none;
+  background: $color-white;
+  border: 0;
+  border-radius: 4px;
+  color: $color-green;
+  font-weight: bold;
+  padding: 7px 10px;
+  width: 150px;
+  z-index: 100;
+
+  &::-ms-expand {
+    display: none;
+  }
+}
+
+option {
+  background: $color-white;
+  border: 2px $color-green solid;
+  padding: 5px 10px;
+}
+
+label {
+  line-height: 29px;
+  margin-left: 70px;
+  position: relative;
+}
+
+.label-select::after {
+  border-bottom: 4px $color-green solid;
+  border-right: 4px $color-green solid;
+  content: "";
+  height: 8px;
+  position: absolute;
+  right: 12px;
+  top: 2px;
+  transform: rotate(45deg);
+  width: 8px;
+}
+
+[type="checkbox"] {
+  visibility: hidden;
+
+  &:disabled + label {
+    opacity: 0.5;
+  }
+
+  + label::before {
+    background: $color-white;
+    border-radius: 4px;
+    content: "";
+    height: 29px;
+    left: -35px;
+    position: absolute;
+    transition: all 0.7s;
+    width: 29px;
+  }
+
+  &:checked + label::after {
+    border-bottom: 4px $color-green solid;
+    border-right: 4px $color-green solid;
+    content: "";
+    height: 10px;
+    left: -25px;
+    position: absolute;
+    top: -1px;
+    transform: rotate(40deg);
+    width: 6px;
+  }
+}
+
+[type="text"] {
+  background: $color-white;
+  border: 0;
+  border-radius: 4px;
+  color: $color-green;
+  font-weight: bold;
+  padding: 7px 10px;
+  text-align: right;
+  width: 60px;
+}
+
+.plant-list {
+  margin: 0 auto;
+  width: 1000px;
+}
+
+.plant-row {
+  border-radius: 5px;
+  margin-bottom: 5px;
+  text-align: left;
+  transition: opacity 0.7s;
+
+  &:hover {
+    transition: background 0.7s;
+  }
+
+  div {
+    display: table-cell;
+    font-size: 12px;
```

Comment:
`font-size: 12px;` are repeated in some classes, so maybe it would be better to put 12px as variable